src/main/java/com/prgrms/springbootbasic/ConsoleApplication.java

```diff
@@ -0,0 +1,56 @@
+package com.prgrms.springbootbasic;
+
+import com.prgrms.springbootbasic.controller.VoucherController;
+import com.prgrms.springbootbasic.domain.Voucher;
+import com.prgrms.springbootbasic.enums.Command;
+import com.prgrms.springbootbasic.enums.VoucherType;
+import com.prgrms.springbootbasic.view.Console;
+import java.util.Map;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component
+@AllArgsConstructor
```

Comment:
private final 사용시 보통 아래 어노테이션을 사용해서 생성자를 만들어줍니다.

```suggestion
@RequiredArgsConstructor
```

Reply:
fianl 필드로 선언된 멤버 변수를 생성자를 만들때는 @RequiredArgsConstrouctor 어노테이션을 사용하는군요!
[18d34da] ConsoleApplication Lombok 어노테이션을 수정했습니다!(@AllArgsConstructor -> @RequiredArgsConstrutor변경)